deb/build

```diff
@@ -46,6 +46,7 @@ fpm -f -s dir -t deb -n "$pkgname" -v "$pkgversion" \
 	--license MIT \
 	--category libs \
 	--deb-changelog "$changelog" \
+	--epoch 1 \
```

Comment:
Maybe add a variable: `epoch=1` and a comment explaining why it has to be changed (link to the issue)?

Reply:
I mean to make sure they epoch for all packages is the same, so then `--epoch $epoch`.